### DISCUSSION TITLE
May 20th 2022 - Visitor Profiles Just Got Better

### DISCUSSION THREAD

**Discussion by jonathonbyrdziak** [merchantprotocol/theadslab#60] [Announcements]
Just rolled out some profile updates to the visitor log. This should now allow you to see into your data and how it associates with all of your visitor records.

This video describes it pretty well
https://www.youtube.com/watch?v=SBsSogvwN_Y

The new updates to the singular data format that I rolled out last week are being seen in this fruit. Basically allows us to quickly connect visitor profiles to their full dataset.

For the next week I'll be working on making this data better, fuller, faster and a higher match rate.

**Comment by abadger13** [comment]
Very cool stuff @jonathonbyrdziak!
